bitnami/kubernetes-event-exporter/values-production.yaml

```diff
@@ -0,0 +1,183 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry, imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+
+## String to partially override kubernetes-event-exporter.fullname include (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override kubernetes-event-exporter.fullname template
+##
+# fullnameOverride:
+
+config:
+  logLevel: error
+  logFormat: json
+  receivers:
+    - name: "dump"
+      file:
+        path: "/dev/stdout"
+        # layout:
+        #   message: "{{ .Message }}"
+        #   reason: "{{ .Reason }}"
+        #   type: "{{ .Type }}"
+        #   count: "{{ .Count }}"
+        #   kind: "{{ .InvolvedObject.Kind }}"
+        #   name: "{{ .InvolvedObject.Name }}"
+        #   namespace: "{{ .Namespace }}"
+        #   component: "{{ .Source.Component }}"
+        #   host: "{{ .Source.Host }}"
+  route:
+    routes:
+      - match:
+          - receiver: "dump"
+        # drop:
+        # - type: "Normal"
+
+rbac:
+  # create specifies whether to install and use RBAC rules.
+  create: true
+
+serviceAccount:
+  # Specifies whether a ServiceAccount should be created
+  create: true
+  # The name of the ServiceAccount to use.  If not set and create is
+  # true, a name is generated using the fullname template
+  name: ""
+
+image:
+  registry: docker.io
+  repository: bitnami/kubernetes-event-exporter
+  tag: 0.9.0-scratch-r1
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+## Pod annotations
+## ref: https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/
+##
+podAnnotations: {}
+
+## SecurityContext holds pod-level security attributes and common container settings.
+##
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  capabilities:
+    add: []
+    drop:
+    - ALL
+  readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 1000
```

Comment:
Could you please change it to the user `1001`? It is the default non-root one used in all our containers/charts.

Reply:
Done